### PR TITLE
Submit vehicle info backend

### PR DESCRIPTION
**Pull request by amichard**
ZEVA-30

Changelog:
- 1 class 1 model
- Frontend BaseController for common functionality like handle input change
- Fixed token renewal from the front-end
- Connected front-end to back-end, but since the fields are just regular input fields
- General code cleanup
- Please note the rename of the capitalization change for named.py
- Updated DB comments

### CODE REVIEW

File: backend/api/models/vehicle.py [hunk 1]
```diff
@@ -1,85 +1,29 @@
 from django.db import models
 
 from auditable.models import Auditable
-from .credit_value import CreditValue
```
Comment (plasticviking) [inline backend/api/models/vehicle.py]: I don't have any issue with moving these into separate files, though I also don't really understand the benefit. The code is pretty tightly-related.

File: backend/api/serializers/vehicle.py [hunk 2]
```diff
@@ -1,38 +1,82 @@
...
+        )
+
 
+class VehicleSerializer(serializers.ModelSerializer):
     credit_value = CreditValueSerializer()
-    type = serializers.SlugRelatedField(slug_field='name', read_only=True)
-    make = serializers.SlugRelatedField(slug_field='name', read_only=True)
-    model = serializers.SlugRelatedField(slug_field='name', read_only=True)
-    trim = serializers.SlugRelatedField(slug_field='name', read_only=True)
-    model_year = ModelYearSerializer()
```
Comment (plasticviking) [inline backend/api/serializers/vehicle.py]: I think `ModelYear` is going to be something more general -- like a compliance period. I'm not sure it makes sense to prepend `Vehicle` to its name.

File: frontend/src/vehicles/components/VehicleListTable.js~master [hunk 3]
```diff
@@ -0,0 +1,108 @@
+/*
```
Comment (plasticviking) [inline frontend/src/vehicles/components/VehicleListTable.js~master]: Leftover merge file?

**Comment by klefler** [comment]
Is this the latest PR in the works? There were some changes that @amichard and I were looking at yesterday afternoon (after this PR was submitted)... and I don't see them here.

File: backend/api/models/vehicle.py (hunk 1 above)
Comment (amichard) [reply backend/api/models/vehicle.py]: It's generally just for organization. Subscribing to the one class per model rule.

**Comment by amichard** [comment]
> Is this the latest PR in the works? There were some changes that @amichard and I were looking at yesterday afternoon (after this PR was submitted)... and I don't see them here.

Sorry, Kristin. I haven't committed them yet. I'll commit them now.

File: backend/api/serializers/vehicle.py (hunk 2 above)
Comment (amichard) [reply backend/api/serializers/vehicle.py]: Sounds good. I'll change it to model_year.py

File: frontend/src/vehicles/components/VehicleListTable.js~master (hunk 3 above)
Comment (amichard) [reply frontend/src/vehicles/components/VehicleListTable.js~master]: It was. Sorry about that. Removing the file.

**Pull request merged**